### ISSUE TITLE
md-icon / accesibility / validation usage with svg (i.e: https://www.google.com/design/icons/) duplicate id's issue

### ISSUE THREAD

**Issue by noamamit92** [angular/material#6796]
When using the same icon twice in a page the html validation fails due to 2 elements having the same ID. 

Since md-icon adds the svg element with the attrbute : md-svg-icon which accepts an id as an input, Is it possible to add the svg element without it's id  - so the validation error wont occur?

a codepen :
 
http://codepen.io/noamamit92/pen/adGjYp#0

notice that if you run the accessibility Audit using your chrome accessibility plugin you will get a duplicate id error for the g element under the svg

**Comment by ThomasBurleson** [comment]
Please provide a CodePen or Plunkr that demonstrates this issue. Here are some starter demo templates that you can use/fork:

*  CodePen template: http://codepen.io/pen?template=bEGJdd
*  Plunkr template: http://plnkr.co/edit/yP0gB9iTacIQb48thud8?p=preview.

**Comment by noamamit92** [comment]
I added a demo:

http://codepen.io/noamamit92/pen/adGjYp#0
